sdk/metricsadvisor/Azure.AI.MetricsAdvisor/CHANGELOG.md

```diff
@@ -23,12 +23,13 @@
   - Renamed class `ServicePrincipalDatasourceCredential` to `ServicePrincipalCredentialEntity`.
   - Renamed class `ServicePrincipalInKeyVaultDatasourceCredential` to `ServicePrincipalInKeyVaultCredentialEntity`.
   - Renamed class `SqlConnectionStringDatasourceCredential` to `SqlConnectionStringCredentialEntity`.
+- Renamed type `DetectionConditionsOperator` to `DetectionConditionOperator`. Also, `MetricWholeSeriesDetectionCondition.CrossConditionsOperator` has been renamed to `ConditionOperator`.
 - Renamed class `MetricAnomalyAlertConfiguration` to `MetricAlertConfiguration`.
-- Renamed class `MetricAnomalyAlertConfigurationsOperator` to `DetectionConditionOperator`. Similarly, `MetricAnomalyAlertConfiguration.CrossMetricsOperator` has been renamed to `ConditionOperator`.
-- Renamed class `DataSourceType` to `DataSourceKind`. Similarly, `GetDataFeedsFilter.SourceType` has been renamed to `SourceKind`, and `DataFeedSource.DataSourceType` has been renamed to `DataSourceKind`.
+- Renamed type `MetricAnomalyAlertConfigurationsOperator` to `MetricAlertConfigurationsOperator`.
```

Comment:
Curious about the rename from `class` to `type`. I see a mix of both now in the changelog and it will be good to have some consistency

Reply:
They are structs. I can use the term `struct` as well, but I wanted users to think they're more like enums.

Reply:
I could call everything type.

Reply:
Maybe better. Not sure how uh user cares about it though